package-lock.json

```diff
@@ -4,11 +4,21 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "app-module-path": {
```

Comment:
there is an awful lot of added  deps.
@Eknir  @ralph-pichler could we have an overview on the security implications?
I would like to see clearly how liberal we can be